server/store/migrations.go

```diff
@@ -68,6 +68,23 @@ var migrations = []Migration{
 			`); err != nil {
 				return errors.Wrapf(err, "failed creating table nodes")
 			}
+
+			if _, err := e.Exec(`
+				CREATE TABLE IF NOT EXISTS edges (
```

Comment:
Do we need to add a unique constraint for the `from_node_id` and `to_node_id` pair?